webapp/components/form/InputChips/InputChips.js

```diff
@@ -0,0 +1,100 @@
+import './InputChips.scss'
+
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Dropdown from '../Dropdown'
+
+import { useInputChips, State } from './store'
+import Chip from './Chip'
+
+const InputChips = (props) => {
+  const {
+    items,
+    itemKey,
+    itemLabel,
+    selection,
+    requiredItems,
+    autocompleteMinChars,
+    readOnly,
+    disabled,
+    validation,
+    onChange,
+    onItemAdd,
+    onItemRemove,
+  } = props
+
+  const { state, Actions } = useInputChips({
```

Comment:
let's be consistent with naming: useLocalState. I also missed that in the autocompleteDialog. you can change both here in this PR

Reply:
Thanks!